docs/pages/access-controls/device-trust/jamf-integration.mdx

```diff
@@ -0,0 +1,303 @@
+```
+
+## Step 3/4. Create a join token
+
+Jamf service requires an MDM token to write devices to Teleport.
+Create the token as shown below:
+
+```code
+$ tctl tokens add --type=mdm
+The invite token: 07c0565aa280595d7645b2964c36ebd1
+This token will expire in 30 minutes.
+
+Use this token to add an MDM service to Teleport.
+
+> teleport start \
+   --token=07c0565aa280595d7645b2964c36ebd1 \
+   --ca-pin=sha256:4b32d9c54b2b3332019d5f0720b8f9a603de03ace07d308bcd743465eee1f200 \
+   --config=/path/to/teleport.yaml
+```
+
+## Step 4/4. Start Jamf service
+
+Using the token created above, start the service:
+
+```code
+$ teleport start --config=/var/lib/teleport.yaml \
+   --token=07c0565aa280595d7645b2964c36ebd1 \
+   --ca-pin=sha256:4b32d9c54b2b3332019d5f0720b8f9a603de03ace07d308bcd743465eee1f200
+```
+
+The initial sync should happen in a few minutes:
+
+```
+2023-06-21T17:26:40-03:00 INFO [JAMF:1]    Jamf service successfully started pid:25757.1 service/service.go:228
+2023-06-21T17:26:40-03:00 INFO [JAMF:1]    Starting sync CutTime:0001-01-01 00:00:00 +0000 UTC FilterRSQL: Mode:1 OnMissing:DELETE pid:25757.1 service/service.go:261
+2023-06-21T17:26:40-03:00 INFO [JAMF:1]    Device sync report, page #0 deletes:0 failures:0 pid:25757.1 upserts:1 service/service.go:666
+2023-06-21T17:26:40-03:00 INFO [JAMF:1]    Sync complete pid:25757.1 service/service.go:277
+```
+
+Using the default configuration, the service will sync devices from Jamf every
+few hours. Once a day a full inventory sync is performed, enumerating all
+devices from Jamf and reflecting any additions or removals on Teleport.
+
+After the initial sync happens, you may verify the synced devices using `tctl
+devices ls`:
+
+```code
+$ tctl devices ls
+Asset Tag    OS      Enroll Status Device ID
+------------ ------- ------------- ------------------------------------
+CXXXXXXXXX17 macOS   not enrolled  20ec6373-9e8e-46e0-8f1c-47ad6b06a768
+CXXXXXXXXX2T macOS   not enrolled  79755778-7cbe-4e2c-83ec-7eaa3d4d7e36
+CXXXXXXXXX3T macOS   not enrolled  665e59d5-393a-4894-841d-edad06329717
+CXXXXXXXXX4T macOS   not enrolled  dd032e90-bfb0-47d5-bce5-e57545f6788f
+CXXXXXXXXX5T macOS   not enrolled  bf189863-a94a-40dc-9013-d96f8dada2f1
+(...)
+```
+
+## Optional: Customize the sync schedule
+
+When using the minimal configuration, described in the steps above, Jamf service
+utilizes a default sync schedule. It is possible to customize sync intervals, as
+well as the set of devices synced from Jamf by applying RSQL filters provided by
+the [Jamf Pro API](
+https://developer.jamf.com/jamf-pro/reference/get_v1-computers-inventory).
+
+The default "inventory" configuration is roughly equivalent to the one below:
+
+```yaml
+jamf_service:
+  enabled: true
+  # ...
+  inventory:
+  - sync_period_partial: 6h
+    sync_period_full: 24h
+    on_missing: DELETE
+    filter_rsql: general.remoteManagement.managed==true
+```
+
+Unpacking the configuration, we have the following:
+
+**sync\_period\_\***
```

Comment:
```suggestion
### `sync\_period`
```

As a general Markdown rule, bold should not be used in place of headers. This comment applies as well to L192 & L217,

Reply:
Done.